automation/taskcluster/decision_task.py

```diff
@@ -159,7 +181,7 @@ def nightly_to_production_app(is_staging, version_name):
 
     build_task_id = taskcluster.slugId()
     build_tasks[build_task_id] = BUILDER.craft_assemble_release_task(
-        variant, 'nightlyLegacy', is_staging, version_name)
+        variant, 'nightly-legacy', is_staging, version_name)
```

Comment:
Note: the "channel" value is used for multiple things, including:
* Secrets
* Index where builds are placed
* Signingscript scope for choosing which key for the product
* Pushapkscript payload value for choosing which deployment config

This channel value must be consistently either `camelCase` or `kebab-case`. This case (`"nightlyLegacy"` was used to identify the secrets) was the only case in which `camelCase` was used. Elsewhere (most notably in signingscript configs), `kebab-case` was used. So, I'm using `kebab-case` across the board for "channel", and refactored this one `camelCase` style to be consistent

Reply:
Thanks for the heads up! Are there any side-effects to this change?

Reply:
Just scope and secret changes, which I have patchsets for elsewhere